README.md

```diff
@@ -1,27 +1,33 @@
-Terraform Syntax Highlighting And Snippets
+Terraform Syntax Highlighting and Snippets
 ==========================================
 
-This [Sublime Text](https://sublimetext.com/) package
-provides basic support for Terraform's [custom `.tf` file type](http://www.terraform.io/docs/configuration/syntax.html),
-along with snippets for each of the basic Terraform resource types.
+This [Sublime Text][] package provides basic support for Terraform's
+[custom `.tf` file type][.tf], along with snippets for each of the basic
+Terraform resource types.
 
-![screenshot](screenshot.png)
+![Syntax highlighting screenshot](screenshot.png)
 
 
 Installation
 ------------
 
 ### Using Package Control
 
-1. Install [Package Control](https://packagecontrol.io/installation), if needed.
-2. Open the palette by pressing `Ctrl+Shift+P` (Win, Linux) or `Cmd+Shift+P` (OS X).
-3. Select _"Package Control: Install package"_
-4. Select _"Terraform"_
+1. Install [Package Control][], if needed.
+2. Open the palette by pressing
+    <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> (Win, Linux) or
+    <kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> (OS X).
+3. Select _Package Control: Install Package_.
+4. Select _Terraform_.
 
 ### Manually
 
-1. Open the Sublime Text Packages folder
-  - OS X: `~/Library/Application Support/Sublime Text/Packages/`
-  - Windows: `%APPDATA%/Sublime Text/Packages/`
-  - Linux (Ubuntu/Debian): `~/.config/sublime-text/Packages/`
-2. Clone this repo
+1. Open the Sublime Text Packages folder:
+    - OS X: `~/Library/Application Support/Sublime Text/Packages/`
```

Comment:
Maybe..

```suggestion
    - MacOS: `~/Library/Application Support/Sublime Text/Packages/`
```

Reply:
Sure. Also above with the command palette?